### DISCUSSION TITLE
how to add part of a.yaml to b.yaml

### DISCUSSION THREAD

**Discussion by caibirdme** [mikefarah/yq#1674] [Q&A] [answered]
a.yaml
```yaml
foo:
  bar: [1,2,3]
```

b.yaml
```yaml
qq:
  we: [4,5,5]
```

I want to achieve `b.yaml/.qq.we += q.yaml/.foo.bar` but don't know how to do that

**Comment by mikefarah** [comment, marked as answer]
You can use the 'load' operator to load content from another file:

```
yq '.qq.we += (load("a.yaml") | .foo.bar)' b.yaml
```

Explanation:
- You want to add '.foo.bar' in file a.yaml to '.qq.we' in b.yaml
- Run yq again b.yaml
- `(load("a.yaml") | .foo.bar)` will load a.yaml and navigate to .foo.bar

**Reply by caibirdme** [reply]
Thx a lot this is really helpful
